airbyte-integrations/connectors/source-salesforce/unit_tests/api_test.py

```diff
@@ -340,12 +340,6 @@ def test_encoding_symbols(stream_config, stream_api, chunk_size, content_type_he
 @pytest.mark.parametrize(
     "login_status_code, login_json_resp, discovery_status_code, discovery_resp_json, expected_error_msg",
     (
-        (
```

Comment:
This has been removed because we fail on rate limit errors [later anyway](https://github.com/airbytehq/airbyte/blob/256c68870e1a49d1607f6b2948d48b54685c6d11/airbyte-integrations/connectors/source-salesforce/source_salesforce/rate_limiting.py#L84-L90). That being said, the reason why we don't retry is unclear to me so I'm willing to change the other part of the code and retry on rate limit errors